livekit-agents/livekit/agents/worker.py

```diff
@@ -358,6 +358,7 @@ async def worker(_: Any) -> web.Response:
                     "agent_name": self._opts.agent_name,
                     "worker_type": agent.JobType.Name(self._opts.worker_type.value),
                     "active_jobs": len(self.active_jobs),
+                    "version": __version__,
```

Comment:
should we also include if it's python or node? or do you already have that info?